scripts/searchSamadi.py

```diff
@@ -0,0 +1,20 @@
+import commands, sys
+
+stringToLookFor = "*v0.1.0+76X_HHAnalysis_2016-04-04.v0"
+output = commands.getoutput("search_SAMADhi.py sample -n %s"%stringToLookFor)
```

Comment:
And also, there's no support for that in `search_SAMADhi.py` (see https://github.com/cp3-llbb/SAMADhi/blob/master/scripts/search_SAMADhi.py), only a command line interface is included, not an API